### ISSUE TITLE
File loading duplication.

### ISSUE THREAD

**Issue by dimamarksman** [metafloor/bwip-js#30]
Steps to reproduce:
1. Create two BWIPJS instaces on the same page.
2. Each instance renders qrcode in the same time.

Result: you'll find in the end of document.head injected script `/bwipp/qrcode.js` **twice**.

I've fixed it on on my side changing canvas.js:
```javascript
// Dynamically load the encoders.
BWIPJS.load = function(path) {
	var script;
	var src = BWIPJS.load.root + path;
	var scriptInjected = Array.prototype.filter.call(document.scripts, function (script) {
		return script.getAttribute('src') === src;
	})[0];

	if (scriptInjected) {
		//Script has been injectd already in document
		return false;
	}
	script = document.createElement('script');
	script.type = 'text/javascript';
	script.src = src;
	document.head.appendChild(script);
};
```

**Comment by metafloor** [comment]
Thanks for the patch.  I am a bit overworked right now.  Hopefully will find time this week to work up a test case for it.

**Comment by metafloor** [comment]
This request has been implemented in version 0.16.0.

**Comment by metafloor** [comment]
I am going to close this issue now since it was implemented in 0.16 and has become a moot point in version 1.0 (no more async loading).
